### ISSUE TITLE
Questions about the data lengths for Figure 2

### ISSUE THREAD

**Issue by chen-yingfa** [HazyResearch/zoology#27]
Hi, thank you very much for this work!

In the arXiv paper regarding Figure 2, it says that the models are trained on sequences with 256 tokens and evaluated on 1024 tokens, however, in the code, it seems that the training data consist of sequences of both 256 in lengths of shorter sequences (64 and 128), and the evaluation data, similarly, consists of sequences of different lengths *up to* 1024. Can you please confirm whether this data mixture is the data used to produce Figure 2?

Thanks in advance.

**Comment by simran-arora** [comment]
Yeah it's up to 256 and up to 1024. The code to produce the plots is here: https://github.com/HazyResearch/zoology/blob/214d4c05ce0f128e633b7013feb8ae92412f0341/zoology/analysis/paper/zoology_figure2.py#L29
